recipes/glib/all/conanfile.py

```diff
@@ -66,7 +70,7 @@ def requirements(self):
         self.requires("libffi/3.3")
         if self.options.with_pcre:
             self.requires("pcre/8.44")
-        if self.options.with_elf:
+        if self.options.get_safe("with_elf", True):
```

Comment:
So you can keep the old version

Reply:
What do you mean?

Reply:
If you apply the previous suggestion, you can revert this change.

Reply:
```suggestion
        if self.options.with_elf:
```